Publishing of 2.4.0 binaries fails

https://github.com/kiwix/kiwix-build/actions/runs/12331239742

> > * In addition the windows ZIP file has somehow a bad filename (is `kiwix-desktop_windows_x64_2.4.zip` but should be ``kiwix-desktop_windows_x64_2.4.0.zip`)
> 
> I believe to have fixed this point with [b1b545a](https://github.com/kiwix/kiwix-build/commit/b1b545a65b416802d04bd5774fef5435b57afb68)

The fix was [wrong](https://github.com/kiwix/kiwix-build/actions/runs/12368720008/job/34532666235):

>     --- creating desktop image ---
>     Traceback (most recent call last):
>       File "D:\a\kiwix-build\kiwix-build\.github\scripts\build_projects.py", line 18, in <module>
>         archive = create_desktop_image(make_release=False)
>                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>       File "D:\a\kiwix-build\kiwix-build\.github\scripts\common.py", line 547, in create_desktop_image
>         build_path = working_dir + ".zip"
>                      ~~~~~~~~~~~~^~~~~~~~
>     TypeError: unsupported operand type(s) for +: 'WindowsPath' and 'str'
>     Error: Process completed with exit code 1.


I corrected it with the second commit in #787.

--- a/.github/scripts/common.py
+++ b/.github/scripts/common.py
@@ -595,7 +595,7 @@ def update_flathub_git():
     call(command)
     shutil.copy(str(BASE_DIR / "org.kiwix.desktop.json"), str(git_repo_dir))
     patch_dir = KBUILD_SOURCE_DIR / "kiwixbuild" / "patches"
-    for dep in ["pugixml"]:
+    for dep in ["pugixml", "libmicrohttpd"]:
         for f in patch_dir.glob("{}_*.patch".format(dep)):
             shutil.copy(str(f), str(git_repo_dir / "patches"))
     command = ["git", "add", "-A", "."]

